packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -161,12 +161,16 @@ - (UIImage *)extractIconFromData:(NSArray *)iconData
                                                        brightness:0.7
                                                             alpha:1.0]];
   } else if ([iconData.firstObject isEqualToString:@"fromAssetImage"]) {
+    // Deprecated: This message handling for 'fromAssetImage' has been replaced by 'asset'.
```

Comment:
Does these need deprecation flag? https://stackoverflow.com/questions/3908715/how-do-i-flag-a-method-as-deprecated-in-objective-c-2-0

Reply:
Tried to find if this kind of pattern is in use but could not find a example so I guess this is okay...